### ISSUE TITLE
[C#] WeaponData crash client

### ISSUE THREAD

**Issue by TuK4z** [altmp/altv-issues#1768]
### Description of the problem

Csharp clientside WeaponData SET crash client without any error in client logs. I tried with damage and RecoilShakeAmplitude
looks like any SET is broken.

### Reproduction steps

`WeaponData wData = Alt.LocalPlayer.GetWeaponData();
if (wData != null)
{
wData.Damage = 10.2f;
 }
else
{
Alt.Log("WeaponData null");
}`

I tried to do it on weaponChange event and another way when player has weapon in hand. Any way client crash on damage SET

### Expected behaviour

Client not to crash

### Operating system

Win 11

### Version

13.48 (release)

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by C0kkie** [comment]
Should be fixed in next dev

**Comment by TuK4z** [comment]
Still crash on 14.01

**Comment by TuK4z** [comment]
I tested this way:
![image](https://user-images.githubusercontent.com/40861802/229303176-9173d8cb-0361-4bef-94c8-38f28ff61470.png)

i ll be back home and i ll try your way. Maybe it's because of WeaponChange event?

**Comment by litolax** [comment]
Reproduced with dev-273

c# client:
```csharp
Alt.OnPlayerWeaponChange += (oldweapon, newweapon) =>
        {
            WeaponChange(oldweapon, newweapon);
        };
        
    public void WeaponChange(uint oldweapon, uint newweapon)
    {
        var wData = Alt.LocalPlayer.GetWeaponData();
        if (wData != null)
        {
            //wData.RecoilShakeAmplitude = 8.2f;
            //wData.Damage = 1;
        }
    }
    ```
    and 
    ```js
    player.giveWeapon("weapon_assaultrifle", 999, true);
    ```
    
    but with pistol weapon I haven't crash
    
    ```js 
player.giveWeapon("weapon_pistol", 999, true);
    ```

**Comment by litolax** [comment]
Should be fixed in 15.0-dev276 and 15.0.104-dev nuget
